docs/source/index.rst

```diff
@@ -81,6 +81,7 @@ Features described in this documentation are classified by release status:
    torch.random <random>
    sparse
    storage
+   torch.testing <testing>
```

Comment:
I'm really not sure where to put this here. It seems kinda alphabetical, so IMO that would be the right place. Feel free to suggest a better position.

Reply:
This seems good